package.json

```diff
@@ -7,15 +7,16 @@
     "type": "git",
     "url": "https://github.com/dai-shi/use-context-selector.git"
   },
+  "browser": "./dist/index.umd.js",
```

Comment:
https://github.com/developit/microbundle#specifying-builds-in-packagejson
This says `"umd:main"`. Should we have both?

Reply:
Hmm, browser is the original field. Probably some bundler that is using umd:main so valid point.